R/add_Layer.R

```diff
@@ -46,6 +64,18 @@ addLayer <- function(mappedData,
     geom = geom,
     geomAttributes = geomAttributes
   )
+
+  # check for geomUnicodeMode
+  geomUnicodeMode <- getOption(
+    x = "ospsuite.plots.GeomPointUnicode",
```

Comment:
please always try to define constants for things like this

Reply:
I do not think to save options as constants make sense. Options are already somethign like options.

Reply:
I meant, define a constant for the **string** `"ospsuite.plots.GeomPointUnicode"` and other constant strings